README.md

```diff
@@ -21,6 +21,7 @@ UAHT може використовуватись згідно прав і сво
 - Контракт: https://polygonscan.com/token/0x0d9447e16072b636b4a1e8f2b8c644e58f3eaa6a#readContract
 - WalletConnect: https://walletconnect.com/
 - UAHT_DAO: https://github.com/starscrowding/UAHT/blob/dev/contracts/UAHT_DAO.sol
+- Whitepaper: https://github.com/OxideDevX/UAHT/blob/dev/public/whitepaper.pdf 
```

Comment:
Please update link to: https://github.com/starscrowding/UAHT/blob/dev/public/whitepaper.pdf